cloud/index.rst

```diff
@@ -6,34 +6,30 @@ Citus Cloud is a fully managed hosted version of Citus Enterprise edition on top
+Citus Cloud plans can vary based on the size of your primary node, size of your distributed nodes, how many distributed nodes and whether you have high availability or not. From within the Citus console you can configure you plan or you can preview what it might look like within the `pricing calculator <https://console.citusdata.com>`_. 
 
-1. A primary instance which you will connect to and is suitable for storing your smaller tables (under 1 million rows) on.
-2. A set of distributed nodes which your distributed tables will be sharded across
-3. High availability, meaning we will be running standbys should you need to fail over. 
+The key items you'll care about for each node:
 
-If you do not need high availability or want a different number or size of nodes than you see available you can create your own custom plan. 
+- Storage - All nodes come with 512 GB of storage
+- Memory - The memory on each node varies based on the size of node you select
+- Cores - The cores on each node varies based on the size of node you select
 
-Custom plans
-------------
-
-Each custom plan allows you to configure the size of your primary instance, the number and size of distributed nodes, and whether the primary instance and nodes have HA enabled. Discounts are available for longer-term commitments on custom plans which make them comparable in price to the preconfigured plans.
+High Availability
+~~~~~~~~~~~~~~~~~
 
+By enabling high availability for your cluster we automatically provision stand-bys. These stand-bys receive streaming updates directly from each of the leader nodes. We continuously monitor the leader nodes to ensure they're available and healthy, in the event of a failure we automatically fail you over. 
 
-Sizing your Citus Cluster
--------------------------
+If you're application needs higher uptime requirements high availability will provide more uptime. Both with and without high availability enabled your data is replicated to S3 so disaster recovery is possible–reducing any risk of data loss. While we do not offer a SLA on uptime; The rough guideline we encourage is that: if an issues occurs and your instance fails and you cannot suffer up to 1 hr of downtime then high availability is encouraged.
 
-All nodes within a Citus cluster come with 512 GB of storage. The number of nodes and size of the nodes you need will vary based on your data volume and performance requirements. We encourage you to focus on the number of logical shards and right distribution key first before focusing on overall size of your cluster. 
-
-Citus will use only as many physical cores to process a query as there are logical shards in your cluster. Thus we recommend creating sufficient shards to give your cluster room to grow. A good estimate is 4-8x the number of cores you currently use in your cluster. For instance choosing 128 logical shards is quite reasonable when you create your distributed tables.
 
+Custom plans
```

Comment:
Putting this next to the `.. raw` confuses Sphinx and causes a dump of the javascript to screen.

![screen shot 2016-09-06 at 3 04 06 pm](https://cloud.githubusercontent.com/assets/911911/18292742/a950b2e0-7443-11e6-91a3-21abb527aba8.png)